app/components/admin/schools/cohort_component.rb

```diff
@@ -20,13 +22,17 @@ def initialize(school:, cohort:, school_cohort:, partnerships_and_relationships:
       end
 
       def training_programme
-        {
-          "core_induction_programme" => "Using DfE-accredited materials",
-          "design_our_own"           => "Designing their own training",
-          "full_induction_programme" => "Working with a DfE-funded provider",
-          "no_early_career_teachers" => "No ECTs this year",
-          "school_funded_fip"        => "School-funded full induction programme",
-        }.fetch(induction_programme_choice, "Not using service")
+        if FeatureFlag.active?(:programme_type_changes_2025)
```

Comment:
I think the logic for the feature flag is replicated here and in the method `training_programme_description`?